packages/eslint-plugin-x/lib/configs/ts.js

```diff
@@ -15,10 +15,10 @@ const syntaxRules = {
   '@typescript-eslint/no-empty-interface': 'off',
   '@typescript-eslint/no-explicit-any': 'off',
   '@typescript-eslint/no-extra-parens': ['error', 'all', { nestedBinaryExpressions: false }],
+  '@typescript-eslint/no-loss-of-precision': 'off',
```

Comment:
IMO we can leave this rule enabled and just refactor the [filters test](https://github.com/empathyco/x/blob/main/packages/x-components/src/utils/__tests__/filters.spec.ts#L94) because at least I don't see a reason for such numbers there.